rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp-submit-document-to-mulesoft-idp.adoc

```diff
@@ -0,0 +1,60 @@
+= Submit Document to MuleSoft IDP 
+
+Send documents to MuleSoft IDP to process them with your published document actions. Each execution generates an Execution ID that you must use when you run the Retrieve Results from MuleSoft IDP action step to query the results of said execution. 
```

Comment:
Flip to put user goal first?

Reply:
The Retrieve Results (something)(action step?)

Do you need the initial cap here and in Execution ID?